### DISCUSSION TITLE
Can't change game_type (CSGO)

### DISCUSSION THREAD

**Discussion by Taxick** [GameServerManagers/LinuxGSM#3834] [General]
Hi :)

I hope you can help me out here.. I'm not allowed to change game mode in csgo

I have try this way: 

1. login with rcon --> `rcon_password mypassword`
2. Try to change gamemode -> `rcon game_mode 1; game_type 2;`
3. Then I restart the server `rcon mp_restartgame 10`

But I get this error: 

```
Can't change replicated ConVar game_type from console of client, only server operator can change its value
L 04/02/2022 - 14:42:10: rcon from "5.186.*.*:29898": command "game_mode 1"
```
I can also change the map/kick/restart game and other things! So i must be logged in with rcon!

But the game mode doesn't work!

If I use a rcon tools I found for windows -> https://sourceforge.net/projects/rconcsgo/

Then I can fine change game mode!

How to fix this?

**Comment by h3o66** [comment]
> Can't change replicated ConVar game_type from console of client, only server operator can change its value
you probably need to do instead:
`rcon game_mode 1; rcon game_type 2;`
the second command probably needs as well a rcon
